sprintly/sprintly-kanban
# Update hapi

#### What does it do?
Updates Hapi and the various plugins we use to their latest versions.

#### Where should the reviewer start?
This is a minor change overall, with a few things worth noting:

1. The custom `redirect_uri` property that we forked Bell in order to have is now supported in the main repo, with a slight tweak to what we already have configured.
2. The call signature of good reporters changed.
3. Since this is a dependency change, `npm-shrinkwrap.json` was updated with `make shrinkwrap`. There appears to be an internal change with how the "from" field worked, so the diff is pretty big.

#### How should this be manually tested?
Remember to `rm -rf node_modules` and `make`. Login / logout should work locally (no need to change config or anything) and pusher forwarding should also still work. Everything else is on the client.
